compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmCachedDeclarations.kt

```diff
@@ -182,6 +186,29 @@ class JvmCachedDeclarations(
             }
         }
 
+    fun getStaticCompanionReplacementDeclaration(jvmStaticFunction: IrSimpleFunction): IrSimpleFunction =
+        staticCompanionReplacementDeclarations.getOrPut(jvmStaticFunction) {
+            val companion = jvmStaticFunction.parentAsClass
+            assert(companion.isCompanion)
+            companion.parentAsClass.makeStaticCopyFunction(jvmStaticFunction)
+        }
+
+    private fun IrClass.makeStaticCopyFunction(target: IrSimpleFunction): IrSimpleFunction = context.irFactory.buildFun {
+        updateFrom(target)
+        name = Name.identifier(context.defaultMethodSignatureMapper.mapFunctionName(target))
+        returnType = target.returnType
+        origin = JvmLoweredDeclarationOrigin.JVM_STATIC_REPLACEMENT_METHOD
+    }.apply proxy@{
```

Comment:
Redundant label.

Reply:
Another case of blindly following the above method unfortunately. I'll fix this too. Thanks.